app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter(post)
+    post.likes_counter = post.likes_counter.to_i + 1
+    post.save!
+  end
```

Comment:
- [ Required ] I think it would be nice if you use the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes_counter` method every time a new `Like` has been saved to the database, then make the update_likes_counter method private. This will ensure that the likes counter is incremented when a new like has been saved to the database, please the suggestion below for an idea of how to achieve this.

## Example code
```ruby
class Like < ApplicationRecord
  belongs_to :user
  belongs_to :post

  after_save :update_likes_counter

  private
  
  def update_likes_counter
    post.update(likes_counter: post.likes.count)
  end
end

```